bndtools.core/bndtools.shared.bndrun

```diff
@@ -8,13 +8,31 @@
   -Xmx2g, \
   -ea
 
+-runpath: \
+	ch.qos.logback.classic,\
+	ch.qos.logback.core,\
+	org.apache.felix.logback,\
+	slf4j.api
+
+-runsystempackages: \
+	org.slf4j;version=1.6.2,\
+	org.slf4j.event;version=1.6.2,\
+	org.slf4j.helpers;version=1.6.2,\
+	org.slf4j.spi;version=1.6.2,\
+	org.slf4j;version=1.7.0,\
+	org.slf4j.event;version=1.7.0,\
+	org.slf4j.helpers;version=1.7.0,\
+	org.slf4j.spi;version=1.7.0
+
 -runproperties: \
 	eclipse.application=org.eclipse.ui.ide.workbench,\
 	eclipse.consoleLog=,\
 	eclipse.product=org.eclipse.sdk.ide,\
 	osgi.console=,\
 	osgi.instance.area.default=../bndtools.test/workspace,\
-	osgi.console.enable.builtin=false
+	osgi.console.enable.builtin=false,\
+	logback.configurationFile=file:${.}/logback.xml,\
```

Comment:
I think you want to use `${fileuri;${.}/logback.xml}` here for our windows friends :-)